src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -2787,6 +2787,8 @@ public function get_token_name() {
 			case self::STATE_FUNKY_COMMENT:
 				return '#funky-comment';
 		}
+
+		return null;
```

Comment:
This would fall through and return "none", which is a type error when the function is annotated to return `?string` (a nullable string). Adding an explicit `null` return fixes this.